pkgs/by-name/tl/tlm/package.nix

```diff
@@ -0,0 +1,27 @@
+{
+  pkgs,
+  lib,
+  fetchFromGitHub,
+  buildGoModule,
+  ollamaVariant ? pkgs.ollama,
```

Comment:
just let it receive ollama using the ollama parameter, that way you can omit pkgs parameter

to change that you can always use a override later

it's basically dependency injection!